### ISSUE TITLE
MNT: change single letter internal variables

### ISSUE THREAD

**Issue by bsipocz** [astropy/astroquery#1572]
E.g. 
* `c` --> `coordinate`
* `r` --> `result`

or something similar, that make sense in the actual context and not extreme abbreviation. This can be done one module at a time and is very much suitable for folks new to the package, to open source, or to python.

**Comment by mfonism** [comment]
@bsipocz I'd love to take this up.

I'll make one commit per module, and when the PR gets accepted I can squash them all into one.

**Comment by bsipocz** [comment]
Sounds like a good plan @mfonism!

Also, given that there are a lot of modules, I would suggest to make a checklist here in the issue to track which ones you have checked and fixed, so you can open a PR before fixing all of them (it also makes it possible for someone else also working on this for a subset of the modules.)

The only tricky bit is making sure that all tests are passing, both local and remote after the changes. (note that there are a bunch of remote ones that are failing, so it's more about making sure there are no new ones failing). 
For that you can run `python setup.py tests -R -P modulename` for the modules you work on. (`-R` stands for remote, if you work on this while not having internet connection, feel free to omit it, we will just be more careful during review).

**Comment by mfonism** [comment]
# Checklist

- [x] alfalfa (#1859)
- [ ] alma
- [ ] astrometry_net
- [ ] atomic
- [ ] besancon
- [ ] cadc
- [ ] casda
- [ ] cds
- [ ] cosmosim
- [ ] dace
- [ ] esa
- [ ] esasky
- [ ] eso
- [ ] exoplanet_orbit_database
- [ ] fermi
- [ ] gaia
- [ ] gama
- [ ] gemini
- [ ] heasarc
- [ ] hitran
- [ ] ibe
- [ ] image_cutouts
- [ ] imcce
- [ ] irsa
- [ ] irsa_dust
- [ ] jplhorizons
- [ ] jplsbdb
- [ ] jplspec
- [ ] lamda
- [ ] lcogt
- [ ] magpis
- [ ] mast
- [ ] mpc
- [ ] nasa_ads
- [ ] nasa_exoplanet_archive
- [ ] ned
- [ ] nist
- [ ] noirlab
- [ ] nrao
- [ ] nvas
- [ ] oac
- [ ] ogle
- [ ] open_exoplanet_catalogue
- [ ] sdss
- [ ] sha
- [ ] simbad
- [ ] skyview
- [ ] solarsystem
- [ ] splatalogue
- [ ] svo_fps
- [ ] template_module
- [ ] tests
- [ ] ukidss
- [ ] utils
- [ ] vamdc
- [ ] vizier
- [ ] vo_conesearch
- [ ] vsa
- [ ] wfau
- [ ] xmatch
- [ ] top level files (in astroquery)

**Comment by mfonism** [comment]
__IMPORTANT__

If you're working on a fix for the aforementioned issue (or plan to work on one) please indicate which module you plan to work on in a comment below. And also, when you're done with your fix, please indicate that it has been fixed, and link to the resulting PR in your comment.

I will update the above checklist accordingly.

This will go a long way to prevent duplication of efforts.

Thank you.

**Comment by nandita11** [comment]
@mfonism I can help with the following packages, if that's okay!

[ ] fermi
[ ] gaia
[ ] gama
[ ] gemini
[ ] heasarc

**Comment by mfonism** [comment]
Sure @nandita11.

I'll update the checklist with your name against the five packages.

**Comment by daisycodes** [comment]
@mfonism I would like to help with the following **hitran, ibe, image_cutouts, imcce, irsa**.

@HarrietAkot I think this might be a good place to help out since you can't find package-novice tickets to work on.

**Comment by HarrietAkot** [comment]
Thanks. I will look into it.

**Comment by mfonism** [comment]
That would be awesome, @daisycodes. I've updated the checklist with your name against the packages you mentioned.

**Comment by sebaucillon** [comment]
@mfonism Hi. I'd like to give xmatch a try. I will get back to you when done to take on more.

**Comment by mfonism** [comment]
> @mfonism Hi. I'd like to give xmatch a try. I will get back to you when done to take on more.

Go ahead, @sebaucillon. I've updated the checklist with your name against `xmatch`. Can you consider taking on `vamdc` to `wfau` when you're done with this one?

**Comment by sebaucillon** [comment]
> > @mfonism Hi. I'd like to give xmatch a try. I will get back to you when done to take on more.
> 
> Go ahead, @sebaucillon. I've updated the checklist with your name against `xmatch`. Could you consider taking on the `v*` and `w*` modules (`vamdc` to `wfau`) when you're done with this one?

Yes I would. I unfortunately haven't figured out how to test astroquery before submitting my pull request to master but I'll get on with `wfau` and `vsa` after that.

**Comment by sebaucillon** [comment]
Hi @mfonism 
I have looked into `xmatch`, `wfau`, `vsa` and `vamdc`.
`vamdc` is clear of this issue.
The others are still in my fork. I have not made a pull request because I do not have a way to test them due to issues with my working environment. You are all welcome to test my changes for me, I'd appreciate it greatly.
Ciao.

**Comment by tinumide** [comment]
Hi @mfonism, I would like to take on the s series, `sdss`, `sha`, `simbad`, `skyview`, `solarsystem`, `splatalogue` and `svo_fps`.
